Automattic/jetpack
# Instant Search: Render interface on search pages

#### Changes proposed in this Pull Request:

* When a user performs a search using a search widget, the search page should be enhanced with the new instant search interface. Note that this feature is gated by a `JETPACK_SEARCH_PROTOTYPE` define.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Yes, this is part of the Jetpack Instant Search project: p3QzjZ-V3.

#### Testing instructions:
* Add `define( "JETPACK_SEARCH_PROTOTYPE", true );` to your wp-config.php.
* Ensure that your site has the Jetpack Pro plan and has Jetpack Search enabled.
* Enter a query into a search widget. 
Alternatively, navigate to a search page like `/?s=this+is+a+search`.
* Ensure that you see a new Instant Search interface replacing the `main` content area on the left side of the page:

<img width="952" alt="Instant_Search_Test" src="https://user-images.githubusercontent.com/4044428/62582566-2ea32a00-b86a-11e9-9e30-d956ab0a5734.png">

#### Proposed changelog entry for your changes:
* No changelog entry necessary; these changes will not be accessible to our users.
